src/lib/ota/jethome.ts

```diff
@@ -0,0 +1,103 @@
+const url = 'https://raw.githubusercontent.com/jethome-ru/zigbee-firmware/master/ota/jethome_ota_index.json';
+import * as common from './common';
+import {Logger, Zh, Ota, KeyValueAny} from '../types';
```

Comment:
Looks like a lot of duplication of `ota.zigbeeOTA`, could you make the url a parameter for `ota.zigbeeOTA`?

Reply:
@adeepn Failed to check if update available for 'Kitchen JetHome WS7' (No image available for imageType '61441')

Reply:
> @adeepn Failed to check if update available for 'Kitchen JetHome WS7' (No image available for imageType '61441') Firmware build date 20231124 Firmware version 12

latest version for ws7 is 12

Reply:
@adeepn Agree, but this is an error message, not that you already have the latest version installed or that there is no newer version. How about making firmware with three event types (single/double/long press) instead of two (single/double)?